integration/helpers/timeouts.go

```diff
@@ -30,9 +30,10 @@ func SetTestTimeouts(t time.Duration) {
 	// TODO(tcsc): Remove this altogether and replace with per-test timeout
 	//             config (as per #8913)
 
-	apidefaults.SetTestTimeouts(t, t)
+	// Randomize the times a little as we don't want to trigger all tasks at the exact same time.
+	apidefaults.SetTestTimeouts(time.Duration(float64(t)*1.0), time.Duration(float64(t)*1.1))
 
-	defaults.ResyncInterval = t
-	defaults.SessionRefreshPeriod = t
-	defaults.HeartbeatCheckPeriod = t
+	defaults.ResyncInterval = time.Duration(float64(t) * 0.9)
+	defaults.SessionRefreshPeriod = time.Duration(float64(t) * 1.2)
+	defaults.HeartbeatCheckPeriod = time.Duration(float64(t) * 1.3)
```

Comment:
```suggestion
	defaults.ResyncInterval = time.Duration(float64(t) * 1.2)
	defaults.SessionRefreshPeriod = time.Duration(float64(t) * 1.3)
	defaults.HeartbeatCheckPeriod = time.Duration(float64(t) * 1.4)
```

so it's never less than the original timeout?